code/game/objects/items/robot/robot_items.dm

```diff
@@ -9,6 +9,7 @@
 	name = "electrically-charged arm"
 	icon_state = "elecarm"
 	var/charge_cost = 30
+	var/stamina_loss_amt = 60
```

Comment:
```suggestion
	var/charge_cost = 30
	///How much damage the attack does per hit. Bypasses armor.
	var/stamina_loss_amt = 60
```
Are you sure you want this to bypass armor?

Reply:
batons do